doc/quick-start/quick-start.md

```diff
@@ -1,5 +1,39 @@
 # Quick Start for Data Prep Kit 
-Here we provided short examples of various uses of the Data Prep Kit.
+Here we provided short examples of various uses of the Data Prep Kit. Most users who want to jump right in can use standard pip install to deploy the data-prep-kit and the python or ray transforms to their virtual python environment. 
+
+- When setting up a virtual environment it is recommended to use python3.11 as in the example below using conda:
+
+    - setup a virtual environment (example using conda) <br>
+        `conda create -n data-prep-kit-1 -y python=3.11`
+<br>
+
+    - activate the new conda environment<br>
+        `conda activate data-prep-kit-1`
+<br>
+
+    - make sure env is swithced to data-prep-kit-1 and Check python version. <br>
+        `python --version`
+        <br> The command above should say: should say : 3.11
+<br>
+
+    - install jupyter lab<br>
+        `pip3 install jupyterlab`   
+<br><br>
+
+- Deploy the latest release of the data prep toolkit library<br>
+    `pip3 install data-prep-toolkit`
+
+or 
+-  deploy the latest releases of the data prep toolkit library and all python transforms
+
+    `pip3 install data-prep-toolkit-transforms`
+
+or 
+-  deploy the latest releases of the data prep toolkit library, all python transforms and all ray transforms
+
+    `pip3 install data-prep-toolkit-transforms-ray`
```

Comment:
ditto

Reply:
This step also requires gcc/g++, while building `fasttext` which is pulled by it.

On conda environemnts, it needs to be installed like:
```
conda install gcc_linux-64
conda install gxx_linux-64
```

Reply:
@shivdeep-singh-ibm do you know if this is specific to Ubuntu? I don't recall having to do that on my MacOS but I also did not use conda.

Reply:
I am using Ubuntu 24.04 that has GCC v13 and can confirm it happens.  Seems like it compiles with GCC v11.  I have a fix here  #550